### PR TITLE
Clear Zest Results Table whenever new Zest script is added

### PR DESCRIPTION
**Pull request by pranavsaxena17**
Fixes zaproxy/zaproxy#601

### CODE REVIEW

**Comment by thc202** [comment]
The changelog needs to be updated.

@psiinon could you clarify what you meant with "opened"? Is that when a script is added or selected?

**Comment by psiinon** [comment]
Oh, good question! It was a long time ago I raised this ;)
I think it was when a new script was added rather than just selected but tbh I'm not completely sure.
I think it could be a pain if you lost the results just because you selected another script, possibly accidentally. But loading a new script seems like a more significant action. 
Shame I didnt add more details at the time :/

**Comment by pranavsaxena17** [comment]
@psiinon Yup, I also thought that it would be easy to lose results by accidentally selecting aonther script.  So, does this need any changes before review?

**Comment by psiinon** [comment]
If thats what your code currently dies then yes. Sorry for the confusion :/

**Comment by pranavsaxena17** [comment]
No, currently it clears the list only when a new script is _added_, as `clear` is invoked inside the `scriptAdded` function, which is called when a script is added.

File: addOns/zest/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Clear Zest Results Panel when new script is opened.
```
Comment (thc202) [inline addOns/zest/CHANGELOG.md:8]: `opened` → `added` to be more clear. The commit/PR message/title could also be tweaked.

Comment (pranavsaxena17) [reply addOns/zest/CHANGELOG.md]: Done

**Comment by thc202** [comment]
Could you amend the message of the first commit? (Commits can be squashed too.)

**Comment by pranavsaxena17** [comment]
@thc202  Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**